uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrl_test.js

```diff
@@ -17,15 +16,15 @@
     'mocks/apis/option-group-api-mock',
     'mocks/apis/public-holiday-api-mock',
     'common/mocks/services/api/contact-mock',
-    'leave-absences/shared/controllers/leave-request-popup-controller',
+    'leave-absences/shared/controllers/sub-controllers/leave-request-ctrl',
```

Comment:
from PR description 

>  It is not directly called and therefore does not contain a separate test file.

we would need to create a concrete child anyway so no point in testing it separately